bokehjs/src/lib/core/dom.ts

```diff
@@ -156,19 +156,22 @@ export function nbsp(): Text {
   return text("\u00a0")
 }
 
+/** @deprecated */
 export function append(element: Node, ...children: Node[]): void {
   for (const child of children) {
     element.appendChild(child)
   }
 }
```

Comment:
Using `@deprecated` syntax in comments like this allows tools like [typedoc](https://typedoc.org/) to produce appropriately annotated documentation. It also allows vscode and other aware editors to mark (usually crossout) usage of such APIs. We can also configure eslint to complain about usage of deprecated APIs.

In the case of internal APIs like these, I don't mind removing them immediately, as long as panel and other holoviz projects are not getting broken. Immediate removal has the benefit of reducing bundle size, because otherwise we would be carrying dead weight.